pages.es/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> Gestiona proyectos de Rust y sus dependencias de módulos (crates).
+> Algunos subcomandos como `cargo build` tienen su propia documentación de uso.
+> Más información: <https://crates.io/>.
+
+- Buscar módulos:
+
+`cargo search {{texto_de_búsqueda}}`
+
+- Instalar un módulo:
+
+`cargo install {{nombre_del_módulo}}`
+
+- Listar módulos instalados:
+
+`cargo install --list`
+
+- Crea un nuevo binario o librería en el directorio actual:
+
+`cargo init --{{bin|lib}}`
+
+- Crea un nuevo binario o librería en el directorio especificado:
+
+`cargo new {{path/to/directory}} --{{bin|lib}}`
+
+- Compila el proyecto de Rust en el directorio actual:
+
+`cargo build`
+
+- Compila el proyecto de Rust en el directorio actual usando el compilador __nightly__:
```

Comment:
The `tldr-pages` clients aren't good working with underscore or bold in the example descriptions
```suggestion
- Compila el proyecto Rust en el directorio actual usando el compilador nightly:
```